### ISSUE TITLE
Charts: Not Accessible

### ISSUE THREAD

**Issue by ZeroBrushV2** [primefaces/primefaces#9725]
### Describe the bug

All the charts generated have no option to add a "aria-label", "fallback text" or specify "aria-describedby".

Without any of this information screen readers will simply not acknowledge the presence of the chart in the page nor describe it function to the user.

For reference:
- https://www.chartjs.org/docs/latest/general/accessibility.html
- https://www.w3.org/WAI/tutorials/images/complex/

### Expected behavior

1. Using the "aria-label" PrimeFaces should generate the following element:

`<canvas id="goodCanvas1" width="400" height="100" aria-label="Hello ARIA World" role="img"></canvas>`
A quick fix should be simply use the title of the chart inside the aria-label, but would be great be able to customize it.

2. Implementation of Fallback should generate the following HTML:

```
<canvas id="okCanvas2" width="400" height="100">
    <p>Hello Fallback World</p>
</canvas>
```
In the current state if you add text or any tag like `<p>` inside the chart component it will generate it outside the canvas, like this:
```

<p>Test</p>
<div class="ui-chart">
	<canvas id="testAC" width="1524" height="437" style="display: block; box-sizing: border-box; height: 349.6px; width: 1219.2px;">
	</canvas>
</div>

```

3. The implementation of "aria-describedby" should check if an element is present in the page with the specified id and insert the attribute. Should generate the following HTML:
```
<canvas id="goodCanvas1" width="400" height="100" aria-describedby="MYDESCRIPTIONID" role="img"></canvas>

....

<p id="MYDESCRIPTIONID">This is a great description for my awesome chart!</p>
```

### PrimeFaces edition

Community

### PrimeFaces version

12.0.0

### JSF implementation

MyFaces

### JSF version

2.0

### Java version

1.8

**Comment by melloware** [comment]
Added new `ariaLabel` property to all the Charts and it will default to the Chart Title if not set.
